keps/sig-api-machinery/2885-server-side-unknown-field-validation/README.md

```diff
@@ -491,6 +515,23 @@ TypedValue](https://github.com/kubernetes/kubernetes/blob/9ff3b7e744b34c099c1405
 object has any unknown or duplicate fields.
 
 
+#### Kubectl Flag
+
+For beta, we will modify the kubectl `--validate` flag from being a bool flag to
+a string flag that accepts the following values:
+
+* `true` or `strict`: If server-side validation is enabled on the server it sends
+  a request with the fieldValidation param set to `Strict`, otherwise it falls
+  back to client-side validation. This is the default
+* `false` or `ignore`: This performs no server-side validation or client-side validation,
+sending a request to the server with fieldValidation param set to `Ignore` if
+server-side validation is enabled.
+* `warn`: This sends a request to the server with the fieldValidation param set
+  to `Warn`. It performs server-side validation, but validation errors are
+  exposed as warnings in the result header rather than failing the request. This
+  will be the default once server-side validation goes GA.
```

Comment:
This will *not* be the default for kubectl, right? Since it's strict today it should continue to be strict.

Reply:
removed